README.md

```diff
@@ -1 +1,72 @@
 ## Unitycatalog function library
+
+The Unitycatalog function library provides convenient APIs to interact with Unitycatalog functions, including functions creation, retrieval and execution.
+The library includes clients for interacting with both Unitycatalog OSS server and Databricks-managed Unitycatalog.
+
+## Installation
+
+```sh
+# install from the production repo
+pip install git+ssh://git@github.com/serena-ruan/unitycatalog-function.git
+```
+> [!NOTE]
+> Once this package is published to PyPI, users can install via `pip install unitycatalog-function`
+
+
+## Get started
+
+### Databricks-managed UC
+To use Databricks-managed Unitycatalog, please follow the [instruction](https://docs.databricks.com/en/dev-tools/cli/authentication.html#authentication-for-the-databricks-cli) to authenticate first and make sure the token has workspace-level privilege for managing UC functions.
+
+#### Prerequisites
+- Install databricks-sdk package with `pip install databricks-sdk`.
+- For accessing the UC functions in Databricks, please create a SQL warehouse following [this instruction](https://docs.databricks.com/en/compute/sql-warehouse/create.html), and save the warehouse id.
+- [Optional] Install databricks-connect package with `pip install databricks-connect`.
+    - This package is only required for creating UC functions using sql body.
```

Comment:
Is `databricks-sdk` lacking this functionality?

Reply:
Ah actually, databricks-sdk supports executing command against a cluster, it doesn't support serverless. Let me add support of using cluster_id with databricks-sdk in a follow-up PR and then update this section :)